zenodo/modules/github/views/badge.py

```diff
@@ -46,22 +46,27 @@
 )
 
 
-def badge(doi):
+def badge(doi, style=None):
     """Helper method to generate DOI badge."""
     doi_encoded = urllib.quote(doi, '')
 
+    if style not in ["flat",
```

Comment:
Can you add a configuration value in ``github/config.py`` where the styles can be defined, so they are not hardcoded.